drivers/SmartThings/zigbee-window-treatment/capabilities/shadeRotateState.yaml

```diff
@@ -0,0 +1,33 @@
+id: stse.shadeRotateState
+version: 1
+status: proposed
+name: shade Rotate State
```

Comment:
can you explain a bit more about what this is and how it's distinct from our other window shade devices that use the `reverse` preference?

Reply:
Hi @greens , it's due to translation support as mentioned at slack